nuxt.config.js

```diff
@@ -34,6 +34,14 @@ export default {
     ]
   },
 
+  // Environment dynamic
+  publicRuntimeConfig: {
+    userRoles: {
+      one: process.env.USER_ROLES_01,
+      two: process.env.USER_ROLES_02
+    }
```

Comment:
Hello, @yoslie I'm just curious what's the difference between `userRoles.one` and `userRole.two`, and why do we need to specify that on the ENV? 🤔

Reply:
due to suggestion from @ayocodingit , we moved it to env variable to avoid injection script..
and to answer the difference between one and two its about roles in user (in the future it will be more than two), for now it's only admin and mitra, wdyt better between using number or kind of user roles as a key? @Ibwedagama

Reply:
the purpose is for security, so we can create alias using uuid or something, but for now there is no alias, will install alias later

Reply:
I see, this is interesting. Thank you for the explanation @yoslie @ayocodingit would love to learn more about this topic 🙏🏼